src/historical_plan_list.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import StarIcon from '@material-ui/icons/Star';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
+import IconButton from '@material-ui/core/IconButton';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import PauseCircleOutlineIcon from '@material-ui/icons/PauseCircleOutline';
+import PlayCircleOutlineIcon from '@material-ui/icons/PlayCircleOutline';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+import { IPlan, IPlanObject } from './queueserver';
+import { RouteComponentProps } from 'react-router-dom';
+import { Box, Card, CardActions, CardContent, Container, Paper, Typography } from '@material-ui/core';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+type Plans = {
+  plans: IPlanObject[];
+}
+
+export class HistoricalPlanList extends React.Component<Plans>{
+
+  handleDelete(uid: string) {
+    alert(uid)
```

Comment:
Why delete/move for history? I think most of the handle methods likely want to go away. We probably want to offer clear history as an option, but history manipulation seems unlikely.

Reply:
It's just left over from a copy-paste, I just removed them.